db/migrate/20160829134746_remove_atomic_container_providers.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveAtomicContainerProviders < ActiveRecord::Migration[5.0]
+  include MigrationHelper
+
+  NAME_MAP = Hash[*%w(
+    ManageIQ::Providers::Atomic::ContainerManager                                 ManageIQ::Providers::Openshift::ContainerManager
```

Comment:
rubocop alerts line too long, but i think it's more readable that adding interpulations or breaking to 2 lines